tests/Integration/Momento.Sdk.Tests/TestCacheClient.cs

```diff
@@ -0,0 +1,374 @@
+    {
+        return ((ICacheClient)client).DictionaryRemoveFieldAsync(cacheName, dictionaryName, field);
+    }
+
+    public Task<CacheDictionaryRemoveFieldsResponse> DictionaryRemoveFieldsAsync(string cacheName, string dictionaryName, IEnumerable<byte[]> fields)
+    {
+        return ((ICacheClient)client).DictionaryRemoveFieldsAsync(cacheName, dictionaryName, fields);
+    }
+
+    public Task<CacheDictionaryRemoveFieldsResponse> DictionaryRemoveFieldsAsync(string cacheName, string dictionaryName, IEnumerable<string> fields)
+    {
+        return ((ICacheClient)client).DictionaryRemoveFieldsAsync(cacheName, dictionaryName, fields);
+    }
+
+    public Task<CacheDictionarySetFieldResponse> DictionarySetFieldAsync(string cacheName, string dictionaryName, byte[] field, byte[] value, CollectionTtl ttl = default)
+    {
+        return ((ICacheClient)client).DictionarySetFieldAsync(cacheName, dictionaryName, field, value, ttl);
+    }
+
+    public Task<CacheDictionarySetFieldResponse> DictionarySetFieldAsync(string cacheName, string dictionaryName, string field, string value, CollectionTtl ttl = default)
+    {
+        return ((ICacheClient)client).DictionarySetFieldAsync(cacheName, dictionaryName, field, value, ttl);
+    }
+
+    public Task<CacheDictionarySetFieldResponse> DictionarySetFieldAsync(string cacheName, string dictionaryName, string field, byte[] value, CollectionTtl ttl = default)
+    {
+        return ((ICacheClient)client).DictionarySetFieldAsync(cacheName, dictionaryName, field, value, ttl);
+    }
+
+    public Task<CacheDictionarySetFieldsResponse> DictionarySetFieldsAsync(string cacheName, string dictionaryName, IEnumerable<KeyValuePair<byte[], byte[]>> elements, CollectionTtl ttl = default)
+    {
+        return ((ICacheClient)client).DictionarySetFieldsAsync(cacheName, dictionaryName, elements, ttl);
+    }
+
+    public Task<CacheDictionarySetFieldsResponse> DictionarySetFieldsAsync(string cacheName, string dictionaryName, IEnumerable<KeyValuePair<string, string>> elements, CollectionTtl ttl = default)
+    {
+        return ((ICacheClient)client).DictionarySetFieldsAsync(cacheName, dictionaryName, elements, ttl);
+    }
+
+    public Task<CacheDictionarySetFieldsResponse> DictionarySetFieldsAsync(string cacheName, string dictionaryName, IEnumerable<KeyValuePair<string, byte[]>> elements, CollectionTtl ttl = default)
+    {
+        return ((ICacheClient)client).DictionarySetFieldsAsync(cacheName, dictionaryName, elements, ttl);
+    }
+
+    public void Dispose()
+    {
+        ((IDisposable)client).Dispose();
+    }
+
+    public Task<FlushCacheResponse> FlushCacheAsync(string cacheName)
+    {
+        return ((ICacheClient)client).FlushCacheAsync(cacheName);
+    }
+
+    public Task<CacheGetResponse> GetAsync(string cacheName, byte[] key)
+    {
+        return QuiesceAndThenRun(() => ((ICacheClient)client).GetAsync(cacheName, key));
+    }
```

Comment:
This is how you quiesce then run some client code. (this is the only one I tweaked)